config/autoload/global.php

```diff
@@ -1,5 +1,8 @@
 <?php
 return [
+    'release-feed' => [
+        'verification_token' => getenv('RELEASE_FEED_TOKEN'),
```

Comment:
I've defined this in our hosting at this point, so it will work both in development (if you have a `.env` file) and in production.